### ISSUE TITLE
Support for dsc file format and net file format

### ISSUE THREAD

**Issue by khalibartan** [pgmpy/pgmpy#514]
@ankurankan what are your views for this.

**Comment by ankurankan** [comment]
@khalibartan Yes we definitely need to support as many file formats as possible. So, if you would like you can start working on this.

**Comment by khalibartan** [comment]
@ankurankan I would, but my end sem exams are just about to begin. If somebody can do this before december then he may, otherwise I'll send a pr for this just after the exams are over.

**Comment by khalibartan** [comment]
- [ ] DSC file format
- [ ] NET file format.

**Comment by khalibartan** [comment]
If someone has not begin working on these issues, then I'm on it. Will send a pr soon.

**Comment by khalibartan** [comment]
@ankurankan Can you find some documentation about specification on these networks .  Example files on [bnlearn repository](http://www.bnlearn.com/bnrepository/) are not generalised files , they have been adjusted ( for eg in .bif format they have removed comments, properties etc.)

**Comment by ankurankan** [comment]
@khalibartan Okay. I will see if I can find something. Also can you look at http://www.cs.cmu.edu/~fgcozman/Research/InterchangeFormat/Old/xmlbif02.html, they have the grammar of BIF format. Just have a look at that and see if you can optimize things in our implementation.

**Comment by khalibartan** [comment]
@ankurankan I have designed the things according to what grammar they have given at that link .. :) .
You can see reference to it in example, https://github.com/pgmpy/pgmpy/pull/512/files#diff-e84ca375a827eebb64245fbfa41835caR33 .

**Comment by ankurankan** [comment]
@khalibartan Cool. And I couldn't get what you meant by: the example files are not generalized. Can you please elaborate ?

**Comment by khalibartan** [comment]
@ankurankan Generalized isn't good term to use actually , my bad because I couldn't find word to express it. If you go through the bnlearn repository the files they don't actually contains all types of grammar that could be present in the model, they have modified these files to remove certain things that aren't necessary for creating models, like property , etc, but can be present in the file . I meant that .

**Comment by khalibartan** [comment]
@ankurankan  I also wanted to ask you something , in .net files I noticed that table arguments were giving, tabular CPDs type of thing directly . I meant the numbers were written within parenthesis , in a hierarchy similar to Tabular CPD (numbers grouped together like in tabular CPD) . Is there a way we can efficiently load these , because there might be ways to work around them really fast , because I'm not able to think it right now, you might get an idea of what I'm talking about if you look at this example
```
potential ( either | lung tub ) 
{
  data = (((1.0 0.0)(1.0 0.0))((1.0 0.0)(0.0 1.0))) ;
}
``` 
If you look at data tag , they are written in a similar fashion to cpd.

**Comment by khalibartan** [comment]
@ankurankan Anything found for this.
